spinnaker_camera_driver/src/nodelet.cpp

```diff
@@ -705,6 +718,8 @@ class SpinnakerCameraNodelet : public nodelet::Nodelet
   bool do_rectify_;  ///< Whether or not to rectify as if part of an image.  Set to false if whole image, and true if in
                      /// ROI mode.
 
+  bool enable_ptp_;
```

Comment:
might have missed this when scrolling through - make sure this is always init'd by the ctor(s).
i'd normally say init here, but looks like that isn't the convention in this code.

Reply:
it's currently initialized on this line: https://github.com/jackg0/flir_camera_driver/blob/gigevision-support/spinnaker_camera_driver/src/nodelet.cpp#L141
I could go through and change the way the drivier inits these variables if you want.